### PR TITLE
[Issue #680] Adds the CDN back

### LINKED ISSUE #680: Implement CloudFront CDN for Static Site
### Summary

We are moving forward with CloudFront as a CDN tool. This work is set up CloudFront for the static site.

### Acceptance criteria

- [ ] CloudFront CDN added to static site
- [ ] Update team documentation

### PR DESCRIPTION
**Pull request by coilysiren**
## Summary

Fixes #680

## Changes proposed

- Uncomments the CDN
- Adds a TLS policy
- Fixes adding the cert to the CDN when applicable

## Testing

I deployed this to both dev (without the cert) and staging (with the cert) to test

### CODE REVIEW

File: infra/modules/service/cdn.tf [hunk 1]
```diff
@@ -1,111 +1,116 @@
...
+      restriction_type = "none"
+    }
+  }
+
+  viewer_certificate {
+    acm_certificate_arn            = var.cert_arn == null ? null : var.cert_arn
+    cloudfront_default_certificate = var.cert_arn == null ? true : false
+    minimum_protocol_version       = local.minimum_protocol_version
+    ssl_support_method             = "sni-only"
+  }
```
Comment (coilysiren) [inline infra/modules/service/cdn.tf:100]: I didn't have this bit fully details before, and it broke a prod deploy

Comment (coilysiren) [reply infra/modules/service/cdn.tf:100]: Everything else is just uncommenting existing work

File: infra/modules/service/cdn.tf [hunk 2]
```diff
@@ -1,111 +1,116 @@
...
-#   #checkov:skip=CKV2_AWS_32:Configure response headers policy in future work
-#   #checkov:skip=CKV_AWS_374:Ignore the geo restriction
-#   #checkov:skip=CKV_AWS_305:We don't need a default root object... we don't need to redirect / to index.html.
-# }
+locals {
+  # We have the option to route the CDN to multiple origins based on the origin id.
+  # We do not currently do this, though.
+  default_origin_id        = "default"
+  ssl_protocols            = ["TLSv1.2"]
+  minimum_protocol_version = "TLSv1.2_2021"
```
Comment (coilysiren) [inline infra/modules/service/cdn.tf:6]: These were chosen randomly on my part

**Review by mdragon — APPROVED** [review]
🚀

**Pull request merged**